data/resume.yml

```diff
@@ -72,96 +76,93 @@ summary_links:
 brief_summary: |
-    Deeply experienced in software design and development methodologies,
-    especially related to the disciplines of software quality, web
-    applications, user experience, and team building. Presently working at the intersection of Rails, GraphQL, and React, developing modern web applications with fast response times.
+    Deeply experienced in software architecture, design, and development methodologies,
+    especially related to the disciplines of web applications, user experience, team
+    building, and processes. Working at the intersection of Rails, GraphQL, and React,
+    developing modern web applications with fast response times.
 skills:
   general:
-    - software engineering
-    - web development
     - business analysis
     - consulting
     - facilitating
+    - software engineering
     - teaching
+    - web development
   software_engineering:
+    - "APIs, REST, GraphQL"
     - architecture
-    - full-stack
     - backend
-    - frontend
-    - low-level design
     - coding
     - debugging
+    - frontend
+    - full-stack
+    - low-level design
     - testing
     - tool chains
-    - "APIs: SOAP, REST, GraphQL"
-  web_frameworks:
-    - Rails
-    - Sinatra
-    - Django
-    - Flask
-    - Express
+  libraries_+_frameworks:
+    - ApolloClient
     - ApolloServer
-    - JAMstack
-    - Gatsby
-  web_libraries:
-    - React
     - GraphQL
-    - ApolloClient
+    - jekyll
+    - middleman
+    - Rails
+    - React
+    - Sinatra
   databases:
     - PostgreSQL
-    - MySQL
-    - SQLite
-    - MongoDB
     - Redis
+    - TimeScaleDB
   languages:
-    - Ruby
+    - C/C++
+    - CSS3
+    - HAML
+    - HTML5
     - JavaScript
+    - PHP
     - Perl
-    - Shell
     - Python 3
-    - PHP
-    - HTML5
-    - CSS3
+    - Ruby
     - SCSS/SASS
-    - HAML
-    - C/C++
+    - Shell
   servers:
+    - AWS ECS, EBS, S3, Cloudfront
     - Netlify
     - nginx
     - puma
-    - AWS ECS, EBS, S3, Cloudfront
-  static_site_generators:
-    - jekyll
-    - middleman
-    - gatsby
+    - unicorn
   tools:
+    - babel
+    - Emacs and Org-mode
+    - make
+    - Miro
     - Node.js
-    - Emacs
-    - Webstorm
-    - RubyMine
+    - Notion
+    - npm/npx
     - rake
+    - RubyMine
+    - Storybook
     - thor
-    - yeoman (yo)
     - webpack
-    - babel
-    - npm/npx
-    - gulp
-    - make
+    - Webstorm
+    - yeoman (yo)
   testing:
+    - Jest
     - Minitest
     - RSpec
-    - Jest
     - Testing Library
+
 methodologies:
-  - Program architecture and design
-  - User-/task-centered design
-  - Test-/behaviour-driven development
-  - Structured analysis, design, development, and testing
-  - Process flow analysis, systems theory, cybernetics
-  - Object-oriented analysis, design, programming, and testing
   - Agile development, scrum master
-  - 'Continuous everything: integration, testing, deployment'
+  - Continuous everything -- integration, testing, deployment
```

Comment:
If I'm looking for CI/CD help, I'm probably scanning each resume for 

> CI/CD

 and might overlook how you specified it here

I liked that you separated out integration and deployment. It shows you understand they are not-the-same-thing.

Reply:
👍🏻